lib/components/isometric_tile_map_component.dart

```diff
@@ -46,13 +45,7 @@ class IsometricTileset {
     final i = tileId % columns;
     final j = tileId ~/ columns;
     final s = size.toDouble();
-    return Sprite.fromImage(tileset, x: s * i, y: s * j, width: s, height: s);
-  }
-
-  /// Load a tileset based on a file name.
-  static Future<IsometricTileset> load(String fileName, int size) async {
```

Comment:
I don't think this method is useless, I think we can provide a different way of loading components without needing to care about first loading images. This is exactly the method I was suggesting to add to the others, and I don't see how leaving this here conflicts at all with the other method we are introducing.

Reply:
I removed to be in the same pattern as the others classes.

I really think that this can't exists anymore, I think that methods like these will confuse people and we will fall on the same problem that we are trying to avoid here.

This is a static method, so this will no have access to the `game.images` instance to make the load, so we will have to use `Flame.images` on those methods, and we will end up on a scenario where some assets are stored on the `game.images`, and some on `Flame.images`, and only `game.images` is automatically cleared. This seems rather confusing for me.

Reply:
We can talk more about this later, gonna solve this for now.